docs/integrations/sumo-apps/index.md

```diff
@@ -26,7 +26,60 @@ Once a request has been submitted, a support ticket is automatically opened. A r
 
 ## Table of Contents
```

Comment:
```suggestion
```

Reply:
@kimsauce  --> I'm sorry - don't know what is wrong here.
The Table of Contents is part of the original index page.
What happens when I click _Commit suggestion_?

Reply:
It will remove this H2 header - that's correct, Commit Suggestion will accept the change.